android/src/main/cpp/NativeProxy.cpp

```diff
@@ -63,6 +63,11 @@ NativeProxy::NativeProxy(
 NativeProxy::~NativeProxy() {
   // removed temporary, new event listener mechanism need fix on the RN side
   // reactScheduler_->removeEventListener(eventListener_);
+
+  // cleanup all animated sensors here, since NativeProxy
+  // has already been destroyed when AnimatedSensorModule's
+  // destructor is ran
+  _nativeReanimatedModule->cleanupSensors();
```

Comment:
Is it only an Android-specific problem? Maybe the destructor of Reanimated module would be a better place for it.

Reply:
It is, it's related to the `NativeProxy` JNI hybrid object being destroyed before the `AnimatedSensorModule`, which doesn't seem to happen with the iOS analogues.